chapters/annotations.tex

```diff
@@ -256,7 +256,8 @@ \subsubsection{Text Markup in Captions}\doublelabel{text-markup-in-captions}
 
 A sequence of one or more newlines (encoded either literally or using the \lstinline!\n!
 escape sequence) means a paragraph break.  (A line break within a paragraph is
-not supported.)
+not supported.)  Any newlines before the first paragraph or after the last paragraph are
+ignored.
```

Comment:
Since we've already defined such a sequence to be some sort of paragraph break, I think we should stick to it:
> (… is not supported, and any paragraph break before the first paragraph or after the last paragraph has no impact.)